docs/spec.md

```diff
@@ -252,6 +252,15 @@ the sever SHALL NOT embedded payload longer than the client-designated maximum.
 }
 ```
 
+## SHLink Direct File Request (with `U` Flag)
+
+When the `U` flag is present, the SHL Receiving Application SHALL retrieve a SHLink's sole encrypted file by issuing a request to the `url` with:
+
+* Method: `GET`
+    * Query parameters
+        * `recipient`: Required. A string describing the recipient (e.g.,the name of an organization or person) suitable for display to the Data Sharer
```

Comment:
I think the answer is the same as for any other SHL request. Whoever manages/hosts the SHL can present the info to the user, or summarize it. For insurance cards, this could mean the patient signs into their Insurer Portal and can see "Your Coverage information has been accessed by..."

Ultimately it's up to the SHL Sharer software to support this, and we can decide if it's realistic or not. But I think the capabilities should be the same with or without the `U` flag.

Reply:
You'd never share a direct link to the payload.url. That URL is useless without the other SHLink metadata (namely: flags + decryption key).

Anyone following the SHL protocol to resolve the link will need dedicate software to retrieve and decrypt the payoad URL. They can bring their own software or if the link has a viewer built in as a prefix (e.g. `https://demo.vaxx.link/viewer#`), that viewer will handle this.

Reply:
Just to add to Josh's note here, this part was just to ensure parity with the existing spec (the 'recipient' key in the POST body) ... I'm hopeful that we would not *require* the server to do anything with it --- but it could be useful for servers to get an aggregate sense of usage of their links.